frontend/src/Indexer/Add/AddIndexerModalContent.js

```diff
@@ -62,8 +62,8 @@ const privacyLevels = [
     value: translate('Private')
   },
   {
-    key: 'semiPublic',
-    value: translate('SemiPublic')
+    key: 'SemiPrivate',
```

Comment:
enum comes from the backend with first char lower.

Reply:
squished